Detect gradle failures

Niko-sk2x opened the pull request:
It's confusing when gradle script fails and remapper says "done" anyway without actually doing anything

LexManos commented on src/main/java/net/minecraftforge/remapper/RemapperGUI.java:
```diff
@@ -288,8 +289,11 @@ void updateGuiState() {
             btnGetModInfo.setEnabled(false);
-        }
-        else {
+        } if (this.buildFailed) {
```
why is this line like this? did you miss a else?

Niko-sk2x replied:
oops, fixing that. I had a bit of a mess trying to split it into 3 commits.

Niko-sk2x committed:
Detect gradle failures

It's confusing when gradle script fails and remapper says "done" anyway without actually doing anything

Niko-sk2x commented:
weird, it merged just fine when I tested it
